components/script/dom/htmllinkelement.rs

```diff
@@ -256,10 +255,22 @@ impl HTMLLinkElement {
 
         let referrer_policy = match self.RelList().Contains("noreferrer".into()) {
             true => Some(ReferrerPolicy::NoReferrer),
-            false => None,
+            false => document.get_referrer_policy(),
+        };
+
+        let request = RequestInit {
+            url: url.clone(),
+            type_: RequestType::Script,
+            destination: Destination::Style,
+            credentials_mode: CredentialsMode::Include,
+            origin: document.url().clone(),
+            pipeline_id: Some(self.global().r().pipeline_id()),
+            referrer_url: Some(document.url().clone()),
+            referrer_policy: referrer_policy,
+            .. RequestInit::default()
```

Comment:
This should set `use_url_credentials` to true per https://html.spec.whatwg.org/multipage/infrastructure.html#create-a-potential-cors-request , as well as set mode to Cors.